app/utils/kcidb.py

```diff
@@ -122,6 +122,20 @@ def push_build(build_id, first, bq_options, db_options={}, db=None):
         })
         bq_data['revisions'] = [bq_revision]
 
+    files = {
+        'kernel_image': build[models.KERNEL_IMAGE_KEY],
+        'modules': build[models.MODULES_KEY],
+        'System.map': build[models.SYSTEM_MAP_KEY],
+    }
+    output_files = []
+    for name, file in files.iteritems():
+        if file:
+            url = "/".join([
+                STORAGE_URL,
+                build[models.FILE_SERVER_RESOURCE_KEY],
+                file])
+            output_files.append({"name": name, "url": url})
```

Comment:
Python tip, this could have been written a bit better using a list comprehension:
```python
base_url = urlparse.urljoin(
    STORAGE_URL, build[models.FILE_SERVER_RESOURCE_KEY])
output_files = list(
    {'name': name, 'url': '/'.join([base_url, file])}
    for name, file in files.iteritems() if file
)
```

Reply:
Yeah, sorry.  I'm not a python expert.  I copied this from the build `output_files` handling.  They should probably both be updated in a follow-up cleanup patch if desired.

Reply:
Alright I'll do that in a follow-up.